packages/apps/plugins/plugin-native/src/NativePlugin.tsx

```diff
@@ -94,3 +115,10 @@ export const safeParseJson = <T extends object>(data: string | undefined | null,
   }
   return defaultValue;
 };
+
+export const NativePlugin = (): PluginDefinition => ({
+  meta,
+  ready: async (plugins) => {
+    void (await initializeNativeApp(plugins));
```

Comment:
```suggestion
    await initializeNativeApp(plugins);
```
`void` is the opposite of `await` so I'm not sure this will actually await it or not. The linter requires `void` in front of unawaited promises just so it's clear that they're promises when reading the code.